### DISCUSSION TITLE
Why has this version of clangd never worked for me?

### DISCUSSION THREAD

**Discussion by ghost** [clangd/clangd#2118] [Q&A] [answered]
I downloaded the latest binary release. When I'm still on Windows, it doesn't work for me. The clangd is included in llvm-mingw or winlibs works. When I switched to Linux, the version of clangd included by the distro was too old, so I downloaded this clangd again. And guess what? It's still not working for me!

Am I setup it wrong? I simply unpacked it and added it to PATH. `clangd` is in PATH. But it failed to provide any code completions.

**Comment by HighCommander4** [comment]
Please post [clangd logs](https://clangd.llvm.org/troubleshooting.html#gathering-logs) and be specific about what is not working.

**Comment by ghost** [comment, marked as answer]
GUI apps don't read the `PATH` environment variable from my `.bashrc`, so `clangd` is not found according to them. This is the case on Linux. The case on Windows is further explored in https://github.com/mstorsjo/llvm-mingw/issues/454.
